### PR TITLE
document memcache usage for listening ot unix socket

### PR DESCRIPTION
**Pull request by kzw**

### CODE REVIEW

File: conf.d/mcache.yaml.example [hunk 1]
```diff
@@ -2,6 +2,7 @@ init_config:
 
 instances:
 #    - url: localhost  # url used to connect to the memcached instance
+#      socket: /socket/path # if url missing; 'dd-agent'user has read permission
```
Comment (olivielpeau) [inline conf.d/mcache.yaml.example]: typo: `'dd-agent'user` -> `'dd-agent' user`

**Comment by olivielpeau** [comment]
Hi @kzw, and thanks for your contribution! Could you address my comment and rebase your branch on top of the master branch of this repo please?

Thanks again!

**Comment by kzw** [comment]
I have fixed the typo and corrected socket permission.

**Comment by irabinovitch** [comment]
@kzw  Thanks for making the corrections, but a rebase / squash are still needed. Thanks!

**Comment by remh** [comment]
Closing in favor of #2431

**Pull request closed without merging**